geomesa-arrow/geomesa-arrow-gt/src/main/scala/org/locationtech/geomesa/arrow/io/DeltaWriter.scala

```diff
@@ -188,16 +175,12 @@ class DeltaWriter(
 
     // set attributes in the vector
```

Comment:
Does this comment need to change?  (Proof that I read things somewhat...;))

Reply:
ah, no i think it's still valid. this foreach block is setting the attributes in the vector, then they get written out to bytes after that.